source/ocsp-support/ocsp-support.rst

```diff
@@ -102,9 +102,11 @@ invalid, the driver SHOULD end the connection.
     attempt to validate the status of the unvalidated certificates
     using the cached CRLs.
 
-7.  If the server’s certificate remains unvalidated and that certificate
-    has an OCSP endpoint, the driver SHOULD reach out to the OCSP
-    endpoint specified and attempt to validate that certificate.
+7.  If the server’s certificate remains unvalidated, that certificate
+    has an OCSP endpoint and ``tlsDisableOcspEndpointCheck`` is true (`if the
```

Comment:
"and ``tlsDisableOcspEndpointCheck`` is true" -> "and ``tlsDisableOcspEndpointCheck`` is false"?

Reply:
Oops, totally! Thanks for the catch! Fixed!